connection.go

```diff
@@ -60,6 +60,7 @@ type Connection struct {
 	protoErrorChan        chan error
 	handshakeFinishedChan chan interface{}
 	doneChan              chan interface{}
+	shutdownFinishedCh    chan struct{}
```

Comment:
For consistency with other similar signaling channels here, please name this `shutdownFinishedChan` and use `chan interface{}` (or `chan any`)

Reply:
https://github.com/blinklabs-io/gouroboros/commit/26d75c059ec39d53ec351a8b767c2b4d66a7b4e6
https://github.com/blinklabs-io/gouroboros/pull/493/commits/25da92c22220431ae2dad904b09c03d08a569ae7